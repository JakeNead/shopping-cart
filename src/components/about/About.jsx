import about from "./about.module.css";

export default function About() {
  return (
    <main>
      <p>About us page</p>
    </main>
  );
}
