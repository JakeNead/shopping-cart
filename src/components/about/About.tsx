import style from "./about.module.css";

export default function About(): JSX.Element {
  return (
    <main className={style.main}>
      <header className={style.header}>
        <h1 className={style.h1}>Our Story</h1>
        <p className={style.photoCredit}>
          Photo by{" "}
          <a href="https://unsplash.com/@goodbetterbetts?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">
            Aaron Betts
          </a>
        </p>
      </header>
      <div className={style.text}>
        <p>
          Lorem ipsum dolor sit, amet consectetur adipisicing elit. Vitae,
          perferendis officia quasi expedita consectetur eveniet maxime quis
          sunt aperiam ipsum totam cumque modi eos minima excepturi. Officiis
          necessitatibus ea repellendus. Veniam quaerat atque officiis dolores.
          Sunt recusandae excepturi dolorum, amet error totam tempore rerum
          officiis blanditiis doloribus dolorem vitae assumenda ullam,
          consequatur molestias debitis esse labore dignissimos, ab veritatis!
          Aut impedit quam aliquid ipsa amet qui quibusdam quia veritatis est
          vitae iure modi laborum exercitationem ea velit cupiditate vel dicta
          ipsam, veniam laboriosam itaque? Temporibus, pariatur unde! Minus vero
          neque laudantium consequuntur molestias animi quas repudiandae
          reprehenderit, alias, ipsam nulla?
        </p>
        <div className={style.buttonContainer}>
          <button className={style.btn}>Link #1</button>
          <button className={style.btn}>Link #2</button>
        </div>
      </div>
    </main>
  );
}
