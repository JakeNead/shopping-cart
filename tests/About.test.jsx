import { render } from "@testing-library/react";
import About from "../src/components/about/About";

describe("About component", () => {
  it("Renders the about page", () => {
    const { container } = render(<About />);
    expect(container).toMatchSnapshot();
  });
});
