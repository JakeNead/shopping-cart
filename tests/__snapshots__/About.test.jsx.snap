// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`About component > Renders the about page 1`] = `
<div>
  <main
    class="_main_400349"
  >
    <header
      class="_header_400349"
    >
      <h1
        class="_h1_400349"
      >
        Our Story
      </h1>
      <p
        class="_photoCredit_400349"
      >
        Photo by
         
        <a
          href="https://unsplash.com/@goodbetterbetts?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash"
        >
          Aaron Betts
        </a>
      </p>
    </header>
    <div
      class="_text_400349"
    >
      <p>
        Lorem ipsum dolor sit, amet consectetur adipisicing elit. Vitae, perferendis officia quasi expedita consectetur eveniet maxime quis sunt aperiam ipsum totam cumque modi eos minima excepturi. Officiis necessitatibus ea repellendus. Veniam quaerat atque officiis dolores. Sunt recusandae excepturi dolorum, amet error totam tempore rerum officiis blanditiis doloribus dolorem vitae assumenda ullam, consequatur molestias debitis esse labore dignissimos, ab veritatis! Aut impedit quam aliquid ipsa amet qui quibusdam quia veritatis est vitae iure modi laborum exercitationem ea velit cupiditate vel dicta ipsam, veniam laboriosam itaque? Temporibus, pariatur unde! Minus vero neque laudantium consequuntur molestias animi quas repudiandae reprehenderit, alias, ipsam nulla?
      </p>
      <div
        class="_buttonContainer_400349"
      >
        <button
          class="_btn_400349"
        >
          Link #1
        </button>
        <button
          class="_btn_400349"
        >
          Link #2
        </button>
      </div>
    </div>
  </main>
</div>
`;
